### PR TITLE
sp.seiga.nicovideo.jp

### LINKED ISSUE #108543: sp.seiga.nicovideo.jp
### Issue URL (Ads)
 [https://sp.seiga.nicovideo.jp/?service_list=](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsp.seiga.nicovideo.jp%2F%3Fservice_list%3D)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #108543

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ m.sport-express.ru##.se-brandedbuttons
 m.ebay-kleinanzeigen.de###srp-sticky
 bakuwaro.com,jiwasoku.com##div[style*="padding: 0px; text-align: center; width: 100%; height: 3"]
 seiga.nicovideo.jp##.ad_320_50
+seiga.nicovideo.jp##.ad_320_100
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:98]: Please replace these two rules with `seiga.nicovideo.jp##.ad` i.e. remove
```
seiga.nicovideo.jp##.ad_320_50
seiga.nicovideo.jp##.ad_320_100
```
and add `seiga.nicovideo.jp` to `m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by Yuki2718** [comment]
Can you also add this domain? https://github.com/AdguardTeam/AdguardFilters/issues/108681
(it's not good to address two issues in a PR but this way conflict can be avoided)

**Comment by SKEIDs** [comment]
> Can you also add this domain to `##.ad`? #108681 (it's not good to address two issues in a PR but this way conflict can be avoided)

i can. and done.

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -789,7 +788,7 @@ nordot.app##.main__adWrapper
 nordot.app##.page__ad:not(.page__ad--discoverMore)
 m.youtube.com##.ytp-ad-action-interstitial
 n-tv.de##.topbanner-mobile
-m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad
+gameranbu.jp ,seiga.nicovideo.jp,m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:791]: Unnecessary white space is inserted here.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: sorry. Fixed it.

**Review by Yuki2718 — APPROVED** [review]

**Comment by SKEIDs** [comment]
@zloyden could you merge this ?

**Review by zloyden — APPROVED** [review]

**Pull request merged**